substrate-archive/src/actors/workers/storage_aggregator.rs

```diff
@@ -51,6 +53,19 @@ where
 		}
 		Ok(())
 	}
+
+	async fn handle_traces(&mut self, ctx: &mut Context<Self>) -> Result<()> {
+		let mut traces = std::mem::take(&mut self.traces);
```

Comment:
why not `let mut traces = std::mem::replace(&mut self.traces, Vec::with_capacity(250));` as it's done for the storage?

I have no clue about why 250 is used but just raising the question because traces and storage are handled different ^^